### PR TITLE
Set default env in scripts

### PR DESCRIPTION
**Pull request by kewu1992**
This PR sets default values for environment variables in
scripts/run_installer.sh and scripts/run_cuda_test.sh.

### CODE REVIEW

**Review by rohitagarwal003 — COMMENTED** [review]
It's unfortunate but we now have these at three places:
here, in the `scripts/gpu-installer-env` file and in the `entrypoint.sh` file.

File: scripts/run_cuda_test.sh [hunk 1]
```diff
@@ -17,16 +17,18 @@
...
 GPU_INSTALLER_ENV_PATH="/etc/gpu-installer-env"
 
 setup() {
   if [ ! -f "${GPU_INSTALLER_ENV_PATH}" ]; then
     /usr/share/google/get_metadata_value "attributes/${GPU_INSTALLER_ENV_KEY}" \
-      > "${GPU_INSTALLER_ENV_PATH}"
+      > "${GPU_INSTALLER_ENV_PATH}" || true
   fi
   source "${GPU_INSTALLER_ENV_PATH}"
+  NVIDIA_INSTALL_DIR_HOST="${NVIDIA_INSTALL_DIR_HOST:-/var/lib/nvidia}"
```
Comment (adityakali) [inline scripts/run_cuda_test.sh]: Move these to top of the file.

File: scripts/run_installer.sh [hunk 2]
```diff
@@ -24,9 +24,14 @@ GPU_INSTALLER_ENV_PATH="/etc/gpu-installer-env"
 setup() {
   if [ ! -f "${GPU_INSTALLER_ENV_PATH}" ]; then
     /usr/share/google/get_metadata_value "attributes/${GPU_INSTALLER_ENV_KEY}" \
-      > "${GPU_INSTALLER_ENV_PATH}"
+      > "${GPU_INSTALLER_ENV_PATH}" || true
   fi
   source "${GPU_INSTALLER_ENV_PATH}"
+  COS_NVIDIA_INSTALLER_CONTAINER="${COS_NVIDIA_INSTALLER_CONTAINER:-gcr.io/cos-cloud/cos-gpu-installer:latest}"
```
Comment (adityakali) [inline scripts/run_installer.sh]: Move these to top of the file.

**Review by adityakali — COMMENTED** [review]
> It's unfortunate but we now have these at three places:
> here, in the scripts/gpu-installer-env file and in the entrypoint.sh file.

No one should rely on whats in `entrypoint.sh` or any other files (they can change). The explicit `scripts/gpu-installer-env` is the only interface that should be used to override any parameters, if needed.

File: scripts/run_cuda_test.sh (hunk 1 above)
Comment (kewu1992) [reply scripts/run_cuda_test.sh]: Done

File: scripts/run_installer.sh (hunk 2 above)
Comment (kewu1992) [reply scripts/run_installer.sh]: Done

**Comment by kewu1992** [comment]
> It's unfortunate but we now have these at three places:
here, in the scripts/gpu-installer-env file and in the entrypoint.sh file.

As Aditya said, we consider entrypoint.sh and scripts/run_installer.sh as implementation details. User should only need to change scripts/gpu-installer-env. Later, even scripts/gpu-installer-env will be wrapped by some more user-friendly interface.

File: scripts/run_cuda_test.sh [hunk 3]
```diff
@@ -17,14 +17,17 @@
 set -o errexit
 set -o nounset
 
-CUDA_TEST_CONTAINER="gcr.io/google_containers/cuda-vector-add:v0.1"
 GPU_INSTALLER_ENV_KEY="cos-gpu-installer-env"
 GPU_INSTALLER_ENV_PATH="/etc/gpu-installer-env"
+# The following environment variables may be changed by cos-gpu-installer-env.
+NVIDIA_INSTALL_DIR_HOST="/var/lib/nvidia"
```
Comment (adityakali) [inline scripts/run_cuda_test.sh:23]: Do you want to comment these out (or even remove) from the `scripts/gpu-installer-env` file?

**Review by adityakali — APPROVED** [review]
Optional comment. LGTM either way.

File: scripts/run_cuda_test.sh (hunk 3 above)
Comment (kewu1992) [reply scripts/run_cuda_test.sh:23]: Done.

**Review by adityakali — APPROVED** [review]
Thanks!

**Pull request merged**